### PR TITLE
WRR-1063: Added a QA sampler for `Spotlight.focus` with `preventScroll` option

### PR DESCRIPTION
**Pull request by 0x64**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
`Spotlight.focus` supports `preventScroll` option from enactjs/enact#3277. It looks better to add a sampler case to test this API in Sandstone.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
A new sampler case is created to compare behaviors with `preventScroll` option and without `preventScroll` option.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRR-1063
enactjs/enact#3277

### Comments
Enact-DCO-1.0-Signed-off-by: Seungcheon Baek (sc.baek@lge.com)

### CODE REVIEW

File: samples/sampler/stories/qa/Spotlight.js [hunk 1]
```diff
@@ -518,6 +518,35 @@ export const FocusRing = () => (
...
+		Spotlight.focus('#FocusWithPreventScrollTarget1');
+	}, []);
+
+	const handleClick2 = useCallback(() => {
+		Spotlight.focus('#FocusWithPreventScrollTarget2', {preventScroll: true});
+	}, []);
+
+	return (
+		<>
+			<p>Click on the first button calls Spotlight.focus() to focus the second button for each scroller.</p>
```
Comment (MikyungKim) [inline samples/sampler/stories/qa/Spotlight.js:532]: I guess it's not working when you "click" by pointer.
So "Click" word makes confusing. Let's reword this to Press 5way OK or enter key?

Comment (0x64) [reply samples/sampler/stories/qa/Spotlight.js:532]: Sure, it looks much better!

File: samples/sampler/stories/qa/Spotlight.module.less [hunk 2]
```diff
@@ -33,3 +33,13 @@
 		transform: scale(1.1);
 	});
 }
+
+.shortPureScroller {
```
Comment (MikyungKim) [inline samples/sampler/stories/qa/Spotlight.module.less:37]: How about just.. `scroller`?

File: samples/sampler/stories/qa/Spotlight.js [hunk 3]
```diff
@@ -518,6 +518,35 @@ export const FocusRing = () => (
...
+	}, []);
+
+	const handleClickPrevented = useCallback(() => {
+		Spotlight.focus('#FocusWithPreventScrollTarget2', {preventScroll: true});
+	}, []);
+
+	return (
+		<>
+			<p>Press OK button on the first button calls Spotlight.focus() to focus the second button for each scroller.</p>
+			Without preventScroll option.
```
Comment (MikyungKim) [inline samples/sampler/stories/qa/Spotlight.js:533]:
```suggestion
			Without preventScroll option
```

File: samples/sampler/stories/qa/Spotlight.js [hunk 4]
```diff
@@ -518,6 +518,35 @@ export const FocusRing = () => (
...
+			<div className={css.shortPureScroller}>
+				<Button className={css.block} onClick={handleClickNotPrevented}>Press OK on this button</Button>
+				<Button className={css.block} id="FocusWithPreventScrollTarget1">To be scrolled</Button>
+			</div>
+			<br />
+			With preventScroll: true option.
```
Comment (MikyungKim) [inline samples/sampler/stories/qa/Spotlight.js:539]:
```suggestion
			With preventScroll: true option
```

File: samples/sampler/stories/qa/Spotlight.module.less (hunk 2 above)
Comment (0x64) [reply samples/sampler/stories/qa/Spotlight.module.less]: 🫡

File: samples/sampler/stories/qa/Spotlight.js (hunk 3 above)
Comment (0x64) [reply samples/sampler/stories/qa/Spotlight.js]: 🫡

File: samples/sampler/stories/qa/Spotlight.js (hunk 4 above)
Comment (0x64) [reply samples/sampler/stories/qa/Spotlight.js]: 🫡

**Review by MikyungKim — APPROVED** [review]
LGTM

**Review by juwonjeong — APPROVED** [review]
LGTM
